### DISCUSSION TITLE
Does ArgoCD work with BitBucket Data Center on AWS

### DISCUSSION THREAD

**Discussion by Arpan-Chakra** [argoproj/argo-cd#5053] [Q&A]
We use BitBucket Data Center on AWS. Will ArgoCD be configurable with it?

**Comment by jannfis** [comment]
As long as your BitBucket exposes Git repositories via HTTPS or SSH with a supported authentication mechanism, it should work fine.

**Reply by Arpan-Chakra** [reply]
Our BitBucket exposed following url
https://bitbucket.cd.private.aws.mebank.com.au/scm/csp/<repo-name>.git
Will that be supported?

**Reply by sbose78** [reply]
This should work @Arpan-Chakra . Could you share if it did?
